README.md

```diff
@@ -0,0 +1,43 @@
+<img src="./logo.png" width="125" height="125" align="right" />
+
+# `__doc__` for [RustPython](https://rustpython.github.io/)
+
+This is the `__doc__` attributes generator for objects written in RustPython.
+
+It's composed of two parts of:
+
+- the `generate_docs.py` script that extracts the `__doc__` attributes from CPython to `docs.inc.rs`
+- the `rustpython-doc` rust crate that uses the `docs.inc.rs` file to create a documentation Database.
+
+This documentation database is then used by macros `pymodule` and `pyclass` macros of the rustpython-derive crate to automatically add the `__doc__` attribute.
+
+The `docs.inc.rs` database file can be generated with
+
+    $ python -m venv gendocs
+    $ source gendocs/bin/activate
+    $ python -I generate_docs.py <path_to_RustPython> docs.inc.rs
```

Comment:
when we use `-I` option, we even don't need a venv.

Reply:
without the venv, I get the error

```
$ python -I generate_docs.py ../RustPython local-no-venv.txt 
JAK v3.5.7
python: /usr/src/debug/pyside-setup-opensource-src-6.3.2/sources/shiboken6/libshiboken/signature/signature_helper.cpp:101: int add_more_getsets(PyTypeObject*, PyGetSetDef*, PyObject**): Assertion `false' failed.
[1]    1404909 IOT instruction (core dumped)  python -I generate_docs.py ../RustPython local-no-venv.txt
```